kit/azure/meshplatform/documentation.tf

```diff
@@ -0,0 +1,20 @@
+output "documentation_md" {
+  value = <<EOF
+# meshStack Integration
+
+meshStack integration sets up this AAD tenant as a meshPlatform.
+To do this we create AAD tenant-level service principals, allowing meshStack to access data
+and orchestrate Azure platform functionality.
+
+## Replicator Service Principal
```

Comment:
a short explanation would help

Reply:
@florianow please describe quickly what each principal does. This should be in the product docs already somewhere. 

If I read the documentation of the cloudfoundation (e.g. as a security auditor) I'm not only interested in what is there (the SPN, name etc.) but also especially _why_ it's there and what it's being used for